src/components/File/File.js

```diff
@@ -0,0 +1,190 @@
+/* ./src/components/{Component}/{Component}.js */
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Icon, MediaObject } from '../..';
+
+const RATIOS = [
+  '1-by-1',
+  '4-by-3',
+  '16-by-9'
+];
+
+export const File = (props) => {
+  const {
+    actions,
+    className,
+    externalIcon,
+    fileType,
+    hideTitle,
+    image,
+    isLoading,
+    ratio,
+    title,
+    ...rest
+  } = props;
+
+  const renderActions = () => {
+    if (!hideTitle) {
```

Comment:
I don't get this. Seems it should be `if (hideTitle)`?

Reply:
the logic is correct, I'll switch the statements tho. That's what I get for renaming `showTitle` to `hideTitle` later in dev